### PR TITLE
Unused Traitor Items: PDA Bombs

### PR DESCRIPTION
**Pull request by Iamgoofball**
Buffs the explosion and removes the RNG "lol u lost" chance.

### CODE REVIEW

**Comment by kevinz000** [comment]
What the FUCK you're making it GIB THE OWNER?

**Comment by Iamgoofball** [comment]
it shouldnt gib

**Comment by kevinz000** [comment]
>Devastation range 1 >shouldn't gib

**Comment by Iamgoofball** [comment]
fixed it

**Comment by kevinz000** [comment]
I don't see how delimbing/outright breaching and killing everyone in a radius of 4-6 tiles is better.

File: code/game/objects/items/devices/PDA/PDA.dm [hunk 1]
```diff
@@ -471,23 +471,8 @@ var/global/list/obj/item/device/pda/PDAs = list()
...
-								difficulty += P.cartridge.access_medical
-								difficulty += P.cartridge.access_security
-								difficulty += P.cartridge.access_engine
-								difficulty += P.cartridge.access_clown
-								difficulty += P.cartridge.access_janitor
-								difficulty += P.cartridge.access_manifest * 2
-							else
-								difficulty += 2
-
-							if(prob(difficulty * 15) || (P.hidden_uplink))
```
Comment (ChangelingRain) [inline code/game/objects/items/devices/PDA/PDA.dm]: >Letting people roundstart explode their target in double agents

File: code/game/objects/items/devices/PDA/PDA.dm [hunk 2]
```diff
@@ -865,9 +850,9 @@ var/global/list/obj/item/device/pda/PDAs = list()
 	if(T)
 		T.hotspot_expose(700,125)
 		if(istype(cartridge, /obj/item/weapon/cartridge/syndicate))
-			explosion(T, -1, 1, 3, 4)
+			explosion(T, -1, 8, 12, 16)
 		else
-			explosion(T, -1, -1, 2, 3)
```
Comment (ChangelingRain) [inline code/game/objects/items/devices/PDA/PDA.dm]: >Larger blast than a minibomb, minus devastation

>you can basically just blow up half the station with people's PDAS alone

**Comment by Archie700** [comment]
>can kill your target from half the station without ever seeing him
>no evidence

how about we take this pr and shove it

**Comment by PKPenguin321** [comment]
Lol

**Pull request closed without merging**